samples/Python/py3.py3

```diff
@@ -0,0 +1 @@
+print("Hello, world!")
```

Comment:
@danilaml We prefer samples from real applications. These samples are used to train the classifier and for our tests.